byoqm/main.py

```diff
@@ -62,12 +62,16 @@ def load(
         )
         exit(1)
     runner: Runner = Runner(
-        quality_model, Path(src_root), Path(output), save_file, language.lower()
+        quality_model,
+        Path(src_root),
+        Path(output),
+        save_file,
+        language.lower(),
     )
     runner.run()
     if show_graphs:
         dashboard = Dashboard(start_date, end_date)
-        dashboard.show_graphs()
+        dashboard.show_graphs(quality_model, src_root)
```

Comment:
IMO we should be consistent with how the graphs displayed can be "configured". Right now, I have to configure the graphs in two different ways depending on what I want to configure:
- For dates I need to configure this when constructing a `Dashboard` object
- For the quality model and source root to show I need to configure this when I call the `show_graphs` method using arguments.

I don't have strong opinions for using the constructor or method call to configure, I just think we need to be consistent.

Reply:
agreed

Reply:
👍

Reply:
We decided to configure as part of the method call not the construction